pep-0581.rst

```diff
@@ -152,6 +166,50 @@ I believe the effort of creating and maintaining GitHub integrations and bots
 is much less than the effort needed to get Roundup up to speed and then to
 continue maintaining it.
 
+Downsides of GitHub
+===================
+
+GitHub is not the perfect issue tracker. Several issues we need to be aware of:
+
+- Fear of uncertainty and vendor lock in. GitHub is proprietary and there is
+  risk of vendor lock-in. However, this is an existing problem since CPython's
+  codebase is already on GitHub. This is also not a unique problem to CPython.
+  As a precautionary step, CPython's repository on GitHub has
+  been backed up on daily basis since June 2018. [#]_
+
+- Bots maintenance costs money, and also takes up volunteer time. We would
+  be shifting maintenance burden from Roundup to the bots. At the very least,
+  so far we've been able to address any bugs/issues related to the bots/GitHub
+  APIs rather quickly, in a matter of days, instead of months or years. GitHub
+  APIs are extensive, and used not by CPython's bots but also by the wider
+  Python community. It makes GitHub APIs more approachable compared to
+  the maintenance of Roundup/bpo.
+
+- Using GitHub could possibly increase the triaging effort. This was first raised
+  as a Zulip topic [#]_, and also brought up during Core Python sprint in
+  September 2018 [#]_. A few solutions have been proposed and considered, such as
+  creating a special triage team [#]_.  After PEP 581's acceptance, GitHub released a
+  new triaging role, currently in beta. The PSF has been in touch with GitHub
+  to have this enabled for Python organization. This is pending GitHub's review [#]_.
+
+- Using GitHub could make it easier for people to post disruptive or spammy comments.
+  It is true that there has been incidents where core developers have to moderate
+  and lock disruptive discussions on GitHub. Thankfully, GitHub interface makes
+  it easy for core developers to moderate discussions. In addition, incidents
+  can be escalated to GitHub.
+
+- Manually editing issue templates can be cumbersome and error-prone. However,
+  for most people, creating issues on GitHub will be a much better experience
+  than creating issue on bpo. The numerous fields and textbox to choose from
+  can be confusing and intimidating to a newcomer, and it was not possible
```

Comment:
was -> is? Or did bpo add the ability to edit messages?

Reply:
"is". I've updated the text. Thanks.